src/main/scala/edu/ie3/simona/model/participant/SystemParticipant.scala

```diff
@@ -154,10 +154,10 @@ abstract class SystemParticipant[CD <: CalcRelevantData](
         KILOVOLTAMPERE
       )
 
-      if (apparentPower.isGreaterThan(sMax)) {
+      if (apparentPower.isGreaterThan(sMax.multiply(1.05))) {
         logger.warn(
           s"The var characteristics \'$qControl\' of model \'$id\' ($uuid) imposes an apparent " +
-            s"power (= $apparentPower) that exceeds " +
+            s"power (= $apparentPower) that exceeds 105 % of " +
```

Comment:
Is that what we want? There seems to be change of behavior between 100% and 105% of sMax.
Until 105% we just deal with having more apparent power than sMax and afterwards we limit it to 100% sMax. So if power actually increases from 105% to 106% we lose power and from 105% to 100%.

Why don't we stick with 100% in the first place? I think exceeding it makes sense for some systems like pvs (where I think we deal with it different anyways) but maybe not for bio mass plants or something like this.

Reply:
Don't know where excatly this comes from. I guess the threshold was increased so that slight exceedings of 100% would not trigger the log statement and the reduction of reactive power.

I'd suggest that I reset this to the way it's handled in dev and we create another issue for this, where we continue this discussion?

Reply:
New issues:  #275

Enhance the issue description, if you see the need.